docs/book/src/developer/architecture/controllers/machine-pool.md

```diff
@@ -96,6 +96,26 @@ The `status` object **may** define several fields that do not affect functionali
 * `failureReason` - is a string that explains why a fatal error has occurred, if possible.
 * `failureMessage` - is a string that holds the message contained by the error.
 
+Example:
+```yaml
+kind: MyMachinePool
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+spec:
+    providerIDList:
+      - cloud:////my-cloud-provider-id-0
+      - cloud:////my-cloud-provider-id-1
+status:
+    ready: true
+```
+
+#### Externally Managed Autoscaler
+
+A provider may implement an InfrastructureMachinePool that is externally managed by an autoscaler. For example, if you are using a Managed Kubernetes provider, it may include its own autoscaler solution. To indicate this to Cluster API, you would decorate the MachinePool object with the following annotation:
+
+`"cluster.x-k8s.io/replicas-managed-by": "external-autoscaler"`
+
+The above annotation indicates to Cluster API that MachinePool replica scaling is not under its enforcement. A practical effect of this is that during autoscaling events (initiated externally, not by Cluster API), when Cluster API observes more or fewer MachinePool replicas, it will update its `Status.Phase` property to the value of `"Scaling"`.
+
```

Comment:
@CecileRobertMichon @jackfrancis 

Regarding:
> All changes that impact providers' adoption of the new release should be announced in the provider updates section of the office hours meeting notes and approved in the PR or issue by both approvers and key affected providers

Should this be raised next week in the office hours?

Reply:
Done